### PR TITLE
Build /cloud page

### LINKED ISSUE #8594: [new /cloud page] build page
- [x] Based on this [brief](https://docs.google.com/document/d/1FBS07dSAdKx6ApC_jQn1IJGoSjYsGCG3xN2JiiVnOZ0/edit#heading=h.meo0azuz09li) and this [copy doc](https://docs.google.com/document/d/1DJZFADvRr_hrHzRWJrUTJxAM2ocZJ_ZWEnQVdKi6JBE/edit), build the /cloud page
- [x] Remove existing redirect from /cloud to /public-cloud
- [x] Add meta image

### PR DESCRIPTION
**Pull request by bethcollins92**
## Done

- Build new `/cloud` page
- Remove existing redirect to `/public-cloud`
- Build and add [meta image](https://assets.ubuntu.com/v1/b0824086-twitter_wide_banner+%288%29.jpeg)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/cloud
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check page against [copy doc](https://docs.google.com/document/d/1DJZFADvRr_hrHzRWJrUTJxAM2ocZJ_ZWEnQVdKi6JBE/edit#heading=h.66vg7hr2fr50) and [design](https://github.com/canonical-web-and-design/ubuntu.com/issues/8593#issuecomment-724596636)

## Issue / Card

Fixes #8594 

## Screenshots

image

### CODE REVIEW

**Comment by anasereijo** [comment]
Looking good 👍 

Just wondering if it should have a secondary nav? Is this a "bubble"?

**Comment by bethcollins92** [comment]
@anasereijo Thanks! This is the only page in the bubble at the minute so I didn't think about the nav, but maybe I should add one still, thanks!

File: templates/cloud/index.html [hunk 1]
```diff
@@ -0,0 +1,251 @@
+{% extends "public-cloud/_base_public-cloud.html" %}
```
Comment (ClementChaumel) [inline templates/cloud/index.html:1]: Shouldn't this be `{% extends "public-cloud/_base-cloud.html" %}`
And do we still need the `public-cloud` folder or can we remove it?

**Review by ClementChaumel — APPROVED** [review]
I've noticed that the _cloud_ and _overview_ links at the top both link to `/cloud` is that intended?
image

Otherwise all it looks good to me! 👍

File: templates/cloud/index.html (hunk 1 above)
Comment (bethcollins92) [reply templates/cloud/index.html:1]: Good spot with the `{% extends "cloud/_base-cloud.html" %}`, thanks! And there has been no mention of removing /public-cloud yet. The _cloud_ and _overview_ are intended to both link to the same page. In the future this bubble may have more pages so there will be more use for the secondary nav.

Comment (ClementChaumel) [reply templates/cloud/index.html:1]: Ok it makes sense! Thanks for explaining!

Comment (pmahnke) [reply templates/cloud/index.html:1]: sorry, I don't think it should have a secondary nav

https://ubuntu.com/tco-calculator

**Comment by bethcollins92** [comment]
@pmahnke secondary nav removed as this page doesn't have a contact us, thanks for clarifying.

**Review by ClementChaumel — APPROVED** [review]
LGTM 👍

**Pull request merged**